src/Database/Adapter/PdoAdapter.php

```diff
@@ -237,14 +238,17 @@ private function createWhere(array $conditions = [], string $where = ''): string
      */
     private function addCondition(string $key, $value): string
     {
-        if (!is_array($value)) {
-            return $this->escapeString($key) . ' = ' . $this->createValue($key, 'where_');
-        }
-        $inConditions = [];
-        foreach (array_keys($value) as $index) {
-            $inConditions[] = $this->createValue($key, 'where_' . $index . '_');
+        $this->splitColumnNameAndOperator($key, $value, $columnName, $operator);
```

Comment:
what about to move initialize to this place (before splitColumnNameAndOperatorCall)?
```
// initialize both column name and operator
$columnName = $key;
$operator = '=';
```

Reply:
I put the initialization inside the function, because it is called in both `addCondition` and `bindCondition` so it would be redundant initialize those variables before `splitColumnNameAndOperator` call

Reply:
Well, I see that adding typehint to the arguments, it breaks.

Probably it's better outside, or maybe `splitColumnNameAndOperator` should return `[string, string]` ([ columnName, operator ]) instead of working on referenced variables.

Reply:
Yes, I wanted to suggest to you something like that - return array with keys columnName and operator and string values instead of using references